### ISSUE TITLE
Some antivirus are blocking "var miner = new CoinHive.Anonymous"

### ISSUE THREAD

**Issue by kixgkqir** [cazala/coin-hive-stratum#160]
Hello,

Please, is it possible to change like this?
var miner = new Something.Anonymous(....

Best!

**Comment by cazala** [comment]
Yes, you can, by following [this guide](https://github.com/cazala/coin-hive-stratum/wiki/Deploy-to-DigitalOcean) and specifically [this part](https://github.com/cazala/coin-hive-stratum/wiki/Deploy-to-DigitalOcean#avoid-adblock)
